isis/CMakeLists.txt

```diff
@@ -186,6 +186,11 @@ endif()
  string(REPLACE ";" " " FLAGS_STR "${thirdPartyCppFlags}")
  set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} ${FLAGS_STR}" )
 
+ if(NOT CMAKE_BUILD_TYPE)
+     set(CMAKE_BUILD_TYPE "RELEASE")
+ endif()
+
+ set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -OG")
```

Comment:
CMake already adds the `-O3` flag when mode=release so only debug mode needed an update.